app/uk/gov/hmrc/exports/models/declaration/submissions/Action.scala

```diff
@@ -16,19 +16,20 @@
 
 package uk.gov.hmrc.exports.models.declaration.submissions
 
-import java.time.LocalDateTime
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 
-import play.api.libs.json.Json
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, Reads, _}
 
-/**
-  * Action class
-  *
-  * @param id unique id of the Action.  This is the converstationId from customs-notifications
-  * @param requestType the request type
-  * @param requestTimestamp timestamp (defaults to now)
-  */
-case class Action(id: String, requestType: RequestType, requestTimestamp: LocalDateTime = LocalDateTime.now())
+case class Action(id: String, requestType: RequestType, requestTimestamp: Instant = Instant.now())
 
 object Action {
-  implicit val format = Json.format[Action]
+  implicit val readLocalDateTimeFromString: Reads[Instant] = implicitly[Reads[LocalDateTime]]
+    .map(_.atZone(ZoneOffset.UTC).toInstant)
```

Comment:
Similar comment

Reply:
This would be the values we have on our database, which are in

`LocalDateTime`

format.
We are fine here